### PR TITLE
WIP Add additional RFC7919 groups

### PR DESCRIPTION
**Pull request by boneyard93501**
Added additional rfc7919 groups and put #[ignore] on expensive test.
we also could add <i>g</i> parameter to the SRG struct and consts.

### CODE REVIEW

**Comment by omershlo** [comment]
awesome work @boneyard93501 !,
 Is this PR ready for review or still WIP?

**Comment by boneyard93501** [comment]
Good to go. We can abstract a bit more when there are additional RFC(s) to be added. thx.

File: src/rfc7919_groups.rs [hunk 1]
```diff
@@ -0,0 +1,308 @@
+// RFC7918 Supported Group Registry
+// https://tools.ietf.org/html/rfc7919#appendix-A
+
+const FFDHE2048_P: &'static str = "FFFFFFFF FFFFFFFF ADF85458 A2BB4A9A AFDC5620 273D3CF1
```
Comment (omershlo) [inline src/rfc7919_groups.rs]: question: in practice why do we need the p value in the code ?

**Comment by boneyard93501** [comment]
we don’t. i’ll take it out.

**Pull request merged**